{"name": "test_successful_order_acceptance", "status": "broken", "statusDetails": {"message": "ValueError: Invalid format specifier ' True' for object of type 'str'", "trace": "self = <allure_pytest.listener.AllureListener object at 0x000001AB848990D0>, item = <Function test_successful_order_acceptance>\n\n    @pytest.hookimpl(hookwrapper=True)\n    def pytest_runtest_setup(self, item):\n        if not self._cache.get(item.nodeid):\n            uuid = self._cache.push(item.nodeid)\n            test_result = TestResult(name=item.name, uuid=uuid, start=now(), stop=now())\n            self.allure_logger.schedule_test(uuid, test_result)\n        yield\n        self._update_fixtures_children(item)\n        uuid = self._cache.get(item.nodeid)\n        test_result = self.allure_logger.get_test(uuid)\n        params = self.__get_pytest_params(item)\n        param_id = self.__get_pytest_param_id(item)\n>       test_result.name = allure_name(item, params, param_id)\n\n.venv\\Lib\\site-packages\\allure_pytest\\listener.py:109: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n.venv\\Lib\\site-packages\\allure_pytest\\utils.py:122: in allure_name\n    return SafeFormatter().format(\nC:\\Users\\iUser\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\string.py:190: in format\n    return self.vformat(format_string, args, kwargs)\nC:\\Users\\iUser\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\string.py:194: in vformat\n    result, _ = self._vformat(format_string, args, kwargs, used_args, 2)\nC:\\Users\\iUser\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\string.py:247: in _vformat\n    result.append(self.format_field(obj, format_spec))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <allure_commons.utils.SafeFormatter object at 0x000001AB8486B3B0>, value = '{ ok}', format_spec = ' True'\n\n    def format_field(self, value, format_spec):\n>       return format(value, format_spec)\nE       ValueError: Invalid format specifier ' True' for object of type 'str'\n\nC:\\Users\\iUser\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\string.py:264: ValueError"}, "start": 1746725942081, "stop": 1746725942081, "uuid": "cfaee663-d549-4580-8e29-1f102c3ba5bf", "historyId": "6adf97f83acf6453d4a6a4b1070f3754", "labels": [{"name": "feature", "value": "Принятие заказа"}, {"name": "parentSuite", "value": "tests.order_test"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "DESKTOP-A83HCPM"}, {"name": "thread", "value": "11540-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.order_test.test_accept_order"}]}